resources/js/package.js

```diff
@@ -100,7 +100,7 @@ function init() {
                 config: window.config,
                 loadingCount: 0,
                 loading: false,
-                loadAutocomplete: false,
+                autocompleteFacadeQuery: '',
```

Comment:
Do we really need this prop to be here on the root Vue instance? Can't we have this on the (loading) autocomplete component?